run_sandmark_backfill.py

```diff
@@ -187,7 +182,7 @@ def check_hash_new(h):
     if args.sandmark_tag_override:
         full_branch_tag = args.sandmark_tag_override
     else:
-        ## TODO: we need to somehow get the '.0' more correctly
+	## TODO: we need to somehow get the '.0' more correctly
```

Comment:
We need to decide if `sandmark_tag_override` is still an optional parameter here. It might be reasonable to make it required and remove the else. Or we could try to infer the version now that we better detection code.

Reply:
With commit 127c30..., a `find_ocaml_version` has now been implemented to return the actual full_branch_tag for the hash.